samples/xml-config/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>io.fabric8</groupId>
     <artifactId>fabric8-maven-parent</artifactId>
-    <version>3.2-SNAPSHOT</version>
+    <version>4.0-SNAPSHOT</version>
```

Comment:
Should this be 4.0-SNAPSHOT?

Reply:
good point. creeped in with the cherry pick. I'm going to fix this right on the branch.